.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Could you add `.vscode/` to .gitignore? And remove this file :)

Reply:
https://stackoverflow.com/questions/1274057/how-to-make-git-forget-about-a-file-that-was-tracked-but-is-now-in-gitignore